src/content/docs/iast/introduction.mdx

```diff
@@ -168,3 +167,32 @@ The following table shows the version 3 (CVSSv3) ratings.
     </tr>
   </tbody>
 </table>
+
+<CollapserGroup>
```

Comment:
I'm don't love these FAQ-style collapsers.

1. I don't think it's a good idea to use our docs site to promise future features.
2. I don't think there's much value in mentioning that we don't have any IAST APIs. The lack of documented IAST APIs communicates that already.
3. I think the note about not using IAST in production belongs in a callout at the top of the intro doc.